### DISCUSSION TITLE
Getting "School Id or Password is not correct." error for newly created schools

### DISCUSSION THREAD

**Discussion by gouravmore** [Sunbird-Saral/Community#77] [Q&A]
Hi,

We have created the schools using the `{{url}}/schools/create` API.
But when we try to log in using this newly created school we are getting the following error in response.

ERROR:

```
{
    "error": "School Id or Password is not correct."
}
```

Can you please guide us in resolving this error?

--cc @coolbung @parthlawate

**Comment by GayatriJadhav14** [comment]
> Hi,
> 
> We have created the schools using the `{{url}}/schools/create` API. But when we try to log in using this newly created school we are getting the following error in response.
> 
> ERROR:
> 
> ```
> {
>     "error": "School Id or Password is not correct."
> }
> ```
> 
> Can you please guide us in resolving this error?
> 
> --cc @coolbung @parthlawate

I'm also facing the same problem. Has anyone solved this issue?

**Reply by GayatriJadhav14** [reply]
I figured out why this error happens. It occurs when user data doesn't get added to the user table. To resolve this, we need to import **const User = require("../models/users")** in school.route.js and include the following code:

const userData = new User({
    userId: req.body.schoolId,
    name: req.body.name,
    schoolId: req.body.schoolId,
    password: req.body.password,
});

await userData.save();
